src/cloudflare/internal/vectorize.d.ts

```diff
@@ -117,15 +119,38 @@ type VectorizeMatch = Pick<Partial<VectorizeVector>, "values"> &
 
 /**
  * A set of vector {@link VectorizeMatch} for a particular query.
+ *
+ * @deprecated This is a legacy version from beta version Vectorize.
  */
 interface VectorizeMatches {
   matches: VectorizeMatch[];
   count: number;
 }
 
+/**
+ * Represents a matched vector for a query along with its score and (if specified) the matching vector information.
+ */
+type VectorizeQueryMatch = Pick<Partial<VectorizeVector>, "values"> &
+  Omit<VectorizeVector, "values"> & {
+    /** The score or rank for similarity, when returned as a result */
+    score: number;
+    /** Indexed metadata associated with the vector. */
+    indexedMetadata?: Record<string, VectorizeVectorMetadata>;
```

Comment:
Are we still maintaining this field? I may have thought wrong, but I understood that we would use a unified `metadata` field in the query match response.

Reply:
Missed this: lot of in flight changes :octopus: 
This should be fixed and "final" now!